### DISCUSSION TITLE
Copy/paste layer-keyframing

### DISCUSSION THREAD

**Discussion by Scvosh** [keiwando/toonsquid#118] [Q&A]
https://user-images.githubusercontent.com/121417238/210160801-bc71e12f-7e95-4672-bcae-383580ddae3e.mp4 

copy paste
When copying a layer, copies the keyframing.
you have to manually delete all the points in order to use the copied layer in some other way.
is it possible to copy a layer without copying keyframing?
I'm from Russia. Therefore, the translation from Google Translate may not be accurate

**Comment by keiwando** [comment]
Thanks for the feedback and the video!
Is this something that you would always want to disable for your workflow or do you often have times where you want the keyframes to be copied and other times where you don't?
I'm essentially wondering whether a simple setting to control this behaviour in the app settings would be enough or not.

**Reply by UndeadLamb** [reply]
I run into this issue as well and I have to the. Go back and delete the keyframes. If you can somehow make it a setting that we can turn off but also turn on while in the same project so we can change the setting. Copy and paste the frames without the keyframes and then turn the setting back in so later on within the same project we CAN copy the keyframes without it tampering the frames that didn't need the keyframes before.

**Comment by Scvosh** [comment]
Yes
  the ability to turn on / off, copying keyframes, when copying a layer.
